include/mbedtls/ssl.h

```diff
@@ -2194,9 +2194,11 @@ void mbedtls_ssl_conf_renegotiation_enforced( mbedtls_ssl_config *conf, int max_
  *                 Lower values can be used to enforce policies such as "keys
  *                 must be refreshed every N packets with cipher X".
  *
+ * \note           When the transport is set to MBEDTLS_SSL_TRANSPORT_DATAGRAM,
+ *                 the maximum renegotiation period is 2^48 - 1.
+ *
  * \param conf     SSL configuration
  * \param period   The threshold value: a big-endian 64-bit number.
- *                 Set to 2^64 - 1 to disable periodic renegotiation
```

Comment:
Perhaps I misunderstood, but I thought that what this documentation is saying is that if I execute something like:
```C
const unsigned char period[] = { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff };
mbedtls_ssl_conf_renegotiation_period(&conf, period);
```
Then the periodic renegotiation whill be disabled. Which I do not think its true. If you check the code for `ssl_check_ctr_renegotiate()` (the function that compares the record counter with the incoming or outgoing data buffers), you will see that in no situation the `ssl->renego_period` is checked to see if it is `2^64 -1` and skips renegotiation.